cmd/storage-rest-client.go

```diff
@@ -268,7 +276,13 @@ func (client *storageRESTClient) ReadAll(volume, path string) ([]byte, error) {
 		return nil, err
 	}
 	defer http.DrainBody(respBody)
-	return ioutil.ReadAll(respBody)
+	buf, err := ioutil.ReadAll(respBody)
+	if err != nil {
+		return nil, err
+	}
+	globalNetStats.Receive(uint64(len(buf)))
+	defer globalNetStats.FinishReceiving(uint64(len(buf)))
```

Comment:
If the err != nil, we would need to cancel the counting that has been initiated - there may be an issue with accuracy here.

Reply:
It doesn't make any sense. If err != nil, we never start counting.

Reply:
Oops, I guess this comment is at the wrong location - what about at other places?

Reply:
Honestly, i cant find a single place this makes sense. Please be more specific